guides/05_custom-components/05_frontend.md

```diff
@@ -283,6 +292,21 @@ export default {
     }
 };
 ```
+Then create a `style.css` file with the following content:
+
+```
+@import "tailwindcss";
+```
+
+Import this file into `Index.svelte`. 
```

Comment:
Let's make it clear here that you need to import the css file inside the script tag not the style (easy mistake to make).